website/content/docs/k8s/operations/server-certificate-rotation.mdx

```diff
@@ -0,0 +1,27 @@
+---
+layout: docs
+page_title: Server Certificate Rotation
+sidebar_title: Server Certificate Rotation
+description: Rotate Server Certificate on Kubernetes Cluster safely
+---
+
+# Rotating Server Certificates
+
+As of Consul Helm version `0.29.0`, if TLS is enabled, new TLS certificates for the Consul Server
+are issued every time the TLS-init job runs. These certificates are signed by the same CA and will
```

Comment:
Should we say on every helm upgrade rather than every time the TLS-init job runs? Or maybe "every time the TLS-init job runs, which happens on every helm upgade"?

I think most folks won't know what that job is.

Reply:
jinx! I've added the same comment 😄